pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>se.sundsvall.messaging</groupId>
     <artifactId>api-service-messaging</artifactId>
-    <version>3.3</version>
+    <version>4.0</version>
```

Comment:
4.1

Reply:
You meant dept44 version? If so it's fixed.

Reply:
Yes! I meant dept44-version :-) 

Everything OK then...